src/cow-react/modules/swap/pure/EthFlow/EthFlowStepper/StatusIcon.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import { Icon } from 'react-feather'
+import styled, { css } from 'styled-components/macro'
+
+export type StatusIconState = 'success' | 'pending' | 'not-started' | 'error'
+
+interface StepIconStyle {
+  bgColor: string
+  color: string
+  borderColor: string
+}
+
+const STATUS_ICON_STYLES: Record<StatusIconState, StepIconStyle> = {
+  success: {
+    bgColor: '#D5E5E3',
+    borderColor: '#D5E5E3',
+    color: '#017B28',
+  },
+  pending: {
+    bgColor: '#F8F9FD',
+    borderColor: '#F8F9FD',
+    color: '#0D5ED9',
+  },
+  'not-started': {
+    bgColor: 'none',
+    borderColor: '#D6DDE9',
+    color: '#D6DDE9',
+  },
+  error: {
+    bgColor: '#f25757',
+    borderColor: '#f25757',
+    color: 'white',
+  },
+}
+
+const LABEL_COLOR: Record<StatusIconState, string> = {
+  success: STATUS_ICON_STYLES['success'].color,
+  pending: STATUS_ICON_STYLES['pending'].color,
+  'not-started': 'gray',
```

Comment:
Why not use the value defined on `STATUS_ICON_STYLES` like the others?

Reply:
cause it was hard to read :) I override some.
@fairlighteth will change colors and styles anyways, so doesn't matter too much